vagrant_repo/environments/splunk_standalone.json

```
@@ -6,6 +6,10 @@
       "apps": {
         "bag": "cerner_splunk/cluster-apps-vagrant:search_head"
       },
+      "logs": {
+        "appender.metrics.maxFileSize": "10000000"
```

Need a comma at the end of this:
```suggestion
        "appender.metrics.maxFileSize": "10000000",
```

---

Ah, thanks https://github.com/cerner/cerner_splunk/pull/191/commits/2ba6438e667355257343e1215d4896b0fa5b85f4